tests/testthat/_snaps/reporter-progress.md

```diff
@@ -315,19 +315,11 @@
     - |     1   0 | reporters/skips                                                 
     \ |     2   0 | reporters/skips                                                 
     v |     2   0 | reporters/skips
-    --------------------------------------------------------------------------------
-    Skip ('reporters/skips.R:2:3'): regular skip
-    Reason: regular skip
-    
-    Skip ('reporters/skips.R:6:3'): skip with details
-    Reason: longer skip:
-    this is what happened
```

Comment:
I guess we don't see details anymore, but that's probably ok. Details are probably more useful for interactive use when you are actually looking at the test in question when debugging something (i.e. you are reminding yourself why you skip this test)